app/controllers/idv/welcome_controller.rb

```diff
@@ -3,18 +3,18 @@ class WelcomeController < ApplicationController
     include IdvStepConcern
     include StepIndicatorConcern
     include StepUtilitiesConcern
+    include GettingStartedAbTestConcern
 
     before_action :confirm_welcome_needed
+    before_action :maybe_redirect_for_getting_started_ab_test
```

Comment:
Did you consider adding this to the agreement controller as well? I don't think it is a must-have but may help keep noise out of the A/B test data

Reply:
Good idea!

Reply:
It will automagically happen because AgreementController checks idv_session.welcome_visited, which won't be set, so it redirects back to Welcome, which redirects to GettingStarted. I'm open to adding it anyway - wdyt?

Reply:
Ahhh, yeah that works!